webui/react/src/services/api-ts-sdk/api.ts

```diff
@@ -3152,6 +3152,199 @@ export const V1FittingPolicy = {
     PBS: 'FITTING_POLICY_PBS',
 } as const
 export type V1FittingPolicy = ValueOf<typeof V1FittingPolicy>
+/**
+ * Flat run respresentation.
+ * @export
+ * @interface V1FlatRun
+ */
+export interface V1FlatRun {
+    /**
+     * The id of the run.
+     * @type {number}
+     * @memberof V1FlatRun
+     */
+    id: number;
+    /**
+     * The time the run was started.
+     * @type {Date | DateString}
+     * @memberof V1FlatRun
+     */
+    startTime: Date | DateString;
+    /**
+     * The time the run ended.
+     * @type {Date | DateString}
+     * @memberof V1FlatRun
+     */
+    endTime?: Date | DateString;
+    /**
+     * The current state of the run(trial).
+     * @type {Trialv1State}
+     * @memberof V1FlatRun
+     */
+    state: Trialv1State;
+    /**
+     * The tags of the run.
+     * @type {string}
+     * @memberof V1FlatRun
+     */
+    tags: string;
```

Comment:
fix: i believe tags should be an array of strings

Reply:
tags in the run table are stored as a json object. In the experiment table they were arrays of strings stored within the config

Reply:
Whoops I just realised that there is no way to add tags/labels to a run at the moment in the same way we do experiments. Instead I have changed it to return the experiment label associated with the run for now. This will also be an array of strings. Renamed to `labels`